### PR TITLE
testnets: Change coral seed node and requesting staking token

### LINKED ISSUE #50: Point testnet references in docs2 to coralnet
Depends on https://github.com/CosmWasm/testnets/issues/20

### PR DESCRIPTION
**Pull request by orkunkl**
Resolves https://github.com/CosmWasm/docs2/issues/50

### CODE REVIEW

File: testnets/testnets.md [hunk 1]
```diff
@@ -20,11 +20,12 @@ Below is the [demonet configuration](https://github.com/CosmWasm/testnets/blob/m
 You can find the other networks configurations at [CosmWasm/testnets](https://github.com/CosmWasm/testnets).
 
 ```sh
-export CHAIN_ID=testing
-export TESTNET_NAME=demo-10
-export RPC=https://rpc.demo-10.cosmwasm.com:443
-export FAUCET=https://faucet.demo-10.cosmwasm.com
-export SEED_NODE=1445f84f409745c554c03557e826edc9757b941a@p2p.demo-10.cosmwasm.com:26656
+export CHAIN_ID=cosmwasm-coral
+export TESTNET_NAME=coral
```
Comment (webmaster128) [inline testnets/testnets.md:22]: Where is this used? The full marketing name is Coralnet

File: testnets/testnets.md [hunk 2]
```diff
@@ -20,11 +20,12 @@ Below is the [demonet configuration](https://github.com/CosmWasm/testnets/blob/m
...
+export RPC=https://rpc.coralnet.cosmwasm.com:443
+export FAUCET=https://faucet.coralnet.cosmwasm.com
+export SEED_NODE=ec488c9215e1917e41bce5ef4b53d39ff6805166@195.201.88.9:26656
+
```
Comment (webmaster128) [inline testnets/testnets.md]: Why this extra newline?

File: testnets/testnets.md (hunk 1 above)
Comment (orkunkl) [reply testnets/testnets.md:22]: It's used in setup scripts. The directory of coral config file in genesis: https://github.com/CosmWasm/testnets/tree/master/coral

Comment (webmaster128) [reply testnets/testnets.md:22]: okay, but here it is coralnet: https://github.com/CosmWasm/testnets/blob/master/coral/defaults.env#L2

Comment (orkunkl) [reply testnets/testnets.md:22]: Apparently we had a miscommunication there. Will not happen again.

Comment (webmaster128) [reply testnets/testnets.md:22]: No worries :) I still wonder if we really need this variable at all but this is out of scope for this PR

Comment (orkunkl) [reply testnets/testnets.md:22]: We can come back to and refactor it on the next testnets.

**Pull request merged**